Microsoft.Toolkit.Uwp.SampleApp/Package.appxmanifest

```diff
@@ -27,22 +27,23 @@
         <uap:SplashScreen Image="Assets\UWPCommunityToolkitSampleAppSplashScreen.png" BackgroundColor="#3750D1" />
       </uap:VisualElements>
       <Extensions>
-        <uap:Extension Category="windows.protocol">
```

Comment:
Same, this change seems unnecessary

Reply:
happened during merging conflicts. Moved to where it was before.